plenum/server/node.py

```diff
@@ -2167,6 +2181,9 @@ def is_query(self, txn_type) -> bool:
     def is_action(self, txn_type) -> bool:
         return txn_type in self.actionReqHandler.operation_types
 
+    def is_txn_writable(self, txn_type):
```

Comment:
The method semantic is not correct as of now.
A correct name here is `can_write`, since it checks whether we can write txns to the pool now in general, and not whether we can write this particular txn.